corehq/apps/registration/views.py

```diff
@@ -424,3 +448,13 @@ def eula_agreement(request):
         current_user.save()
 
     return HttpResponseRedirect(request.POST.get('next', '/'))
+
+
+@transaction.atomic
+@login_required
+def send_desktop_reminder(request):
+    if request.method == 'POST':
+        send_mobile_experience_reminder(request.user.username,
```

Comment:
Nitpicky, but you could just pass the couch user and then in the function derive both the username and full name from that.

Reply:
When I only need 2 properties, I generally prefer to just pass them to make it very clear what is being used. Beyond 2, I agree with you though.